src/script/Monitor/ReportRecallQuota.py

```diff
@@ -29,18 +30,11 @@ def readpwd():
 def createQuotaReport(rucioClient=None, account=None):
     """
     create a dictionary with the quota report to be sent to MONIT
-    even if we do not report usage at single RSE's now, let's collect that info as well
-    returns {'rse1':bytes, 'rse':bytes,..., 'totalTB':TBypte}
+    we do not report usage at single RSE's now, we don't collect that info either
```

Comment:
i suggest to remove this comment which was specific of using get_local_account_usage